commit 9a6688c584085597cbbeaa1fd2289eecb6a44561 (refs/changes/00/900/1)
Author: David Pacheco <dap@joyent.com>
Date:   2016-11-14T15:47:29-08:00 (2 years, 11 months ago)
    
    MANTA-3029 use git+https url in marlin README for "npm install"

diff --git a/README.md b/README.md
index c3e7047..33a068f 100644
--- a/README.md
+++ b/README.md
@@ -39,7 +39,7 @@ bits to build on OS X.  To use these tools:
 
 or just:
 
-    $ npm install git+ssh://git@github.com:joyent/manta-marlin.git
+    $ npm install git+https://github.com/joyent/manta-marlin.git
 
 At this point, you'll have several tools available inside the "sbin" directory:
 
